src/main/scala/beam/utils/mapsapi/hereapi/HereService.scala

```diff
@@ -40,16 +36,17 @@ class HereService(adapter: HereAdapter) {
             Segment(
               coordinates = coordinates,
               lengthInMeters = span.lengthInMeters,
-              speedLimitInKph = span.speedLimitInKph
+              speedLimitInMetersPerSecond = span.speedLimitInMetersPerSecond
             )
         }
       }
   }
 
-  case class TmpSpan(startIndex: Int, endIndex: Int, lengthInMeters: Int, speedLimitInKph: Option[Int])
+  case class TmpSpan(startIndex: Int, endIndex: Int, lengthInMeters: Int, speedLimitInMetersPerSecond: Option[Int])
 }
 
 object HereService {
+  private val hereTimeout: Duration = Duration("15 seconds")
```

Comment:
Why bump from 5 sec to 15 seconds?

Reply:
it is possible to do batch queries. just o be "sure" it will not get timeout errors